### PR TITLE
Added libs to classpaths

### PR DESCRIPTION
**Pull request by Shamanou**
I added the libraries to the classpath to enable codacy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/d122f3edbb6e4ebfa8c74c4b219ce3a2)](https://www.codacy.com/app/Shamanou/FAIRifier_2?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=Shamanou/FAIRifier&amp;utm_campaign=Badge_Grade)
```
Comment (kburger) [inline README.md]: Is the codacy badge linked to Shamanou/FAIRifier_2 or to DTL-FAIRData/FAIRifier? Ideally it should be added to the DTL-FAIRData one. We can discuss handling forks and badges after a standup if need be.

**Review by kburger — COMMENTED** [review]
Changes look good, but do think about the badges for a bit.

File: README.md (hunk 1 above)
Comment (Shamanou) [reply README.md]: aaah ok i have indeed 2 different badges for the two good you caught that one

**Comment by kburger** [comment]
Minor issue as reported by codacy: a repeated semicolon on an import. Could you fix that for the next commit?

**Review by kburger — APPROVED** [review]

**Pull request merged**